hw/ip/edn/doc/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "EDN HWIP Technical Specification"
+---
+
+# Overview
+
+This document specifies EDN hardware IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}})
+
+
+## Features
+
+The Entropy Distribution Network (EDN) block provides both hardware and software interfaces to the [CSRNG IP]({{< relref "hw/ip/csrng/doc" >}}) module.
+Simpler hardware interfaces are provided so that a peripheral block does not have to directly interface with the CSRNG module.
+- A set of registers is provided for firmware to manage a CSRNG application port.
+- Hardware interfaces support four request/acknowledge interfaces, or can be configured to act as a broadcast type of interface (only 1 of 4 interfaces).
```

Comment:
do i understand this statement correctly as in..

- there are 4 HW interfaces
- one of the 4 HW interfaces can be configured as broadcast

Reply:
do you think we should consider making the number of interfaces higher? 
I ask because... assume we have 4 low bandwidth peripherals who consumed all 4 interfaces.  Does this mean if there's a 5th peripheral, we'd need both another EDN and another CSRNG instance?

Because I think the req/ack interfaces are likely going to come from blocks that are not going to request frequently, but want very clean looking entropy whenever they do.  So it feels like in that scenario CSRNG may be able to support more than 4. 

Let me know what you think, or if I misunderstood.

Reply:
Correct on what the 4 interfaces do now - but this is a stake in the ground. EDN is flexible to handle more ports. More EDNs handle more CSRNG instances. If the desire is more ports and less instances (doable), then EDN can be changed to allow it fairly easily. However, it cannot be done through parameters unfortunately since we cannot have both flexible number of ports and flexible width of data sizes. But can pick one or the other, so flexible from that perceptive.

Reply:
sounds good. Then let's revisit if needed during integration.

Reply:
Assuming no action required.